moveit_simple/src/moveit_simple.cpp

```diff
@@ -230,6 +324,48 @@ bool Robot::getPose(const std::vector<double> & joint_point,
 
 
 
+bool Robot::getPose(const std::vector<double> & joint_point,
+                    const std::string& custom_tool_frame,
+                    Eigen::Affine3d & pose) const
+{
+  std::lock_guard<std::recursive_mutex> guard(m_);
+  
+  bool GET_POSE_FLAG = false;
```

Comment:
Variable names should be `lower_case`.  Don't append the word `flag`.   It's redundant with the data type, which the IDE displays for you.